virttest/qemu_vm.py

```diff
@@ -1043,13 +1043,13 @@ def set_value(opt_string, key, fallback=None):
             elif optget("spice_port") != "no":
                 set_value("port=%s", "spice_port")
 
-            set_value("password=%s", "spice_password", "disable-ticketing")
+            set_value("password=%s", "spice_password", "disable-ticketing=on")
```

Comment:
As the comment in the bz, for qemu6.0, it will show: `'password' option is deprecated and insecure, use 'password-secret' instead`.

Reply:
@vivianQizhu  This option is different with others, a simple replacement can not work with the old version.

Reply:
@yanan-fu Ah, thanks for the info, let me check.

Reply:
@yanan-fu Okay, it should be `--object secret,id=vncsec0,file=passwd.txt  --spice port=5901,password-secret=vncsec0` and only apply to the new qemu, I think it is a quite different one, not belong to this `short-form` issue, do you agree if I fix it with another patch?

Reply:
I see the new issue already be created, it is ok for me,  we can fix it with another patch.